pymc_marketing/mmm/base.py

```diff
@@ -377,7 +377,7 @@ def compute_channel_contribution_original_scale(self) -> DataArray:
             coords=channel_contribution.coords,
         )
 
-    def plot_contribution_curves(self) -> plt.Figure:
+    def plot_direct_contribution_curves(self) -> plt.Figure:
```

Comment:
If this warranted a more specific name, perhaps it also warrants a descriptive docstring?